day_7/fizzbuzz.rb

```diff
@@ -0,0 +1,28 @@
+(1..100).each do |ele|
+  if  ele % 3 == 0 && ele % 5 == 0
+    puts 'FizzBuzz'
+  elsif ele % 3 == 0
+    puts 'Fizz'
+  elsif  ele % 5 == 0
+    puts 'Buzz'
+  else
+    puts ele
+  end
+end
+
+
+#bonus
+puts "Enter any number"
+num = gets.chomp.to_i
+
+num.times do |ele|
```

Comment:
what would be a better name for this block variable?

Reply:
number